### ISSUE TITLE
Attribute mapping

### ISSUE THREAD

**Issue by jaredpalmer** [RafalFilipek/jsxfromhtml#6]
```js
function Img(props) {
  console.log(props);
  return <img {...props} />;
}
```

will log this out:

```
{ 'data-tag': 'img',
  class: '....',
  src: '....',
  alt: '.... }
```
However, `class` is being fully passed to JSX, which throws...
```
warning.js?8a56:36 Warning: Unknown DOM property class. Did you mean className?
    in img (at Page.js:38)
    in Img (created by ToJSX)
    in div (created by ToJSX)
    in span (created by ToJSX)
    in div (created by ToJSX)
```

**Comment by RafalFilipek** [comment]
Yep. I forgot to mapAttributes [here](https://github.com/RafalFilipek/jsxfromhtml/blob/master/src/step.js#L38) (just like [here](https://github.com/RafalFilipek/jsxfromhtml/blob/master/src/step.js#L48).

I've just released 1.1.0 - can you check it?

**Comment by jaredpalmer** [comment]
lgtm
